backend/core/detection.py

```diff
@@ -298,6 +298,7 @@ def handle_bgp_update(self, message: Dict) -> NoReturn:
 
             if not self.redis.exists(
                     monitor_event['key']) or 'hij_key' in monitor_event:
+                self.redis.set(monitor_event['key'], '', ex=60 * 60)
```

Comment:
this function is always executed at the end so we can move it at the start and save time from the potential RTT